packages/create-codes/src/helpers/omit.ts

```diff
@@ -0,0 +1,15 @@
+type Options = {
+  key?: string;
+  value?: string;
+};
+
+export function omit(
+  dependencies: Record<string, string>,
+  { key, value }: Options
+) {
+  const filteredEntries = Object.entries(dependencies)
+    .filter(([k]) => (key ? ![key].includes(k) : true))
+    .filter(([, v]) => (value ? ![value].includes(v) : true));
```

Comment:
[Q] It looks like the `key` and `value` must be an exact match to be filtered out. In the case where the given value is `4.17.21` and the dependency version is `^4.17.21`, should it be ignored?

I was thinking you could normalize the `value` (remove `^` or `~` if it exists) before `.filter()` and just check the version number.